.changeset/light-facts-fall.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+<!-- START components/app-header -->
+`AppHeader` - Fixed classname so that non-active controls in the AppHeader will be styled as disabled in small screens when the side nav is expanded
```

Comment:
Would it make more sense to say:

"`AppHeader` - Fixed classname so that non-active controls in the AppHeader will be styled as disabled in when the side nav is expanded and in overlay mode"

?

Reply:
yeah, I think your suggestion is better. I might be explicitly say "…when the `AppSideNav` is expanded…" as well